packages/appeal-reply-service-api/api/openapi.yaml

```diff
@@ -0,0 +1,94 @@
+openapi: 0.0.1
+info:
+  title: Appeal Reply Service API Documentation
+  description:
+    API to <ul><li>create, retrieve and modify appeal replies</li><li>
+  version: 1.0.3
+  license:
+    name: MIT
+    url: 'https://opensource.org/licenses/MIT'
+servers:
+  - url: 'http://localhost:3000/'
```

Comment:
This _probably_ wants to be changed to the port assigned in Docker Compose

Reply:
Done